packages/gatsby/src/utils/develop-proxy.ts

```diff
@@ -12,6 +14,17 @@ interface IProxyControls {
 
 const noop = (): void => {}
 
+const adminFolder = path.join(
+  path.dirname(require.resolve(`gatsby-admin`)),
+  `public`
+)
+
+const serveAdmin = st({
+  path: adminFolder,
+  url: `/___admin`,
```

Comment:
This might be a limitation of the `st` package, but I do like that the `/___graphql` path is pattern-based and forgiving for typos. Have you taken that into consideration already?

Reply:
Have not taken that into consideration, good call! From a quick glance, it does not look like the `st` package supports this. I tried some other packages and they didn't quite work, so I'd rather ship this for now and fix that later?